src/EnergyPlus/SolarShading.cc

```diff
@@ -676,7 +676,7 @@ namespace SolarShading {
 #else
                 auto error_callback = [](const int messageType, const std::string & message, void * /*contextPtr*/){
                     if (messageType == Pumbra::MSG_ERR) {
-                        ShowFatalError(message);
+                        ShowSevereError(message);
```

Comment:
@mbadams5 I'm worried that this won't effectively handle problems within Penumbra.